### PR TITLE
Consume dev-site props-table-loader

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Testing the new custom react-docgen based props-table-loader from terra-dev-site

### CODE REVIEW

File: packages/terra-clinical-onset-picker/src/terra-dev-site/doc/clinical-onset-picker/ClinicalOnsetPicker.1.doc.jsx [hunk 1]
```diff
@@ -40,6 +41,7 @@ const DocPage = () => (
       {
         componentName: 'Onset Picker',
         componentSrc: OnsetPickerSrc,
```
[the remark this answers is not in the document]
Comment (bjankord) [reply packages/terra-clinical-onset-picker/src/terra-dev-site/doc/clinical-onset-picker/ClinicalOnsetPicker.1.doc.jsx]: Updated in this commit: https://github.com/cerner/terra-clinical/pull/566/commits/c1c70ca1faa79a84a314d93a5eb95c516a6d1e7a

File: packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/clinicalDataGrid.1.doc.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import DocTemplate from 'terra-doc-template';
 import ReadMe from '../../../../docs/README.md';
 import { name } from '../../../../package.json';
 // Component Source
-import DataGridSrc from '!raw-loader!../../../../src/DataGrid';
+import DataGridPropsTable from '!terra-props-table-loader!../../../../src/DataGrid';
```
Comment (emilyrohrbough) [inline packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/clinicalDataGrid.1.doc.jsx]: @mjhenkes @bjankord   Should this be 
```diff
import DataGridPropsTable from '!terra-props-table-loader!../../../../src/DataGrid';
import DataGridPropsTable from ../../../../src/DataGrid?'terra-props-table-loader;
```
And should this be loading from `lib`?

Comment (mjhenkes) [reply packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/clinicalDataGrid.1.doc.jsx]: not yet, the resource query will be applied with mdx dev site and no, we want this to load from source.

**Comment by bjankord** [comment]
Closing this in favor of a follow up PR once terra-dev-site supports MDX

**Pull request closed without merging**